File: expenses-tracker-server/src/main/kotlin/com/toptal/ggurgul/timezones/domain/services/TimeService.java
package com.toptal.ggurgul.timezones.domain.services;

import org.springframework.stereotype.Component;

import java.io.Serializable;
import java.util.Date;

@Component
public class TimeService implements Serializable {

    private static final long serialVersionUID = -3301695478208950415L;

    public Date now() {
        return new Date();
    }
}

File: expenses-tracker-auth/src/main/kotlin/com/ggurgul/playground/extracker/auth/services/TimeService.kt
package com.ggurgul.playground.extracker.auth.services

import org.springframework.stereotype.Component

import java.io.Serializable
import java.util.Date

@Component
class TimeService : Serializable {

    fun now(): Date {
        return Date()
    }

    companion object {

        private const val serialVersionUID = -3301695478208950415L
    }
}
